### DISCUSSION TITLE
Cue sheet ;  How to describe path to the music file?

### DISCUSSION THREAD

**Discussion by maple4062** [LMS-Community/slimserver#1128] [General]
when cue sheet is in the same directory with the relevant music file, LMS can interpretate the cue sheet. It is fine.
But I cannot see how to write path to the music file when cue sheet is in the different directory from the directory of music file.

Assuming that the NAS has the directory tree as follow.
                                 - /Cue_file ---- /Classical ------- symphony1.cue 
  /mnt  --- /Music ----/OrgData ---- /Classical ------- symphony1.wav

(OS is UBUNTU and Nas is mounted in the /mnt directory with the name "Music")
 
I wrote the cue sheet with path to the music file, such as
"FILE "/mnt/Music/OrgData/Classical/symphony1.wav" WAVE".
This path did not work.

Your advice is highly appreciated.

**Comment by michaelherger** [comment]
You better ask this question in https://forums.slimdevices.com.

**Reply by maple4062** [reply]
I will do so. Thank you!
